yocto/build-testexport.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# vagrant-cookbook
```

Comment:
Typo

Reply:
where exactly?

Reply:
Sorry, thought it was the name of the script.